### PR TITLE
Added my caption

### PR DESCRIPTION
**Pull request by michaelwreeves**
# My Pull Request
This resolves issue #2 

## Other items
- Here is one item
- Here is another item

## Checklist
- [x] To do thing 1
- [x] To do thing 2

### CODE REVIEW

**Comment by michaelwreeves** [comment]
## Another comment
Here is another comment
💯

**Review by slcardinal — COMMENTED** [review]
Michael, this looks good to me.

**Review by nickscape — APPROVED** [review]
Looks awesome!

**Review by jimbyer — APPROVED** [review]

File: _slides/14-michaelwreeves.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Here is my caption
```
Comment (michaelwreeves) [inline _slides/14-michaelwreeves.md:6]: Here is a suggestion:
```suggestion
A better caption
```

Comment (forbushbl) [inline _slides/14-michaelwreeves.md:6]:
```suggestion
Here is a different caption
```

**Review by forbushbl — APPROVED** [review]

**Pull request merged**